projects/README.md

```diff
@@ -0,0 +1,21 @@
+# Welcome to Projects of MMPose
+
+In this folder, we welcome all contribution of keypoint detection techniques from community.
+
+Here, these requirements, e.g. code standards, are not that strict as in core package. Thus, developers from the community can implement their algorithms much more easily and efficiently in MMPose. We appreciate all contributions from community to make MMPose greater.
+
+Here is an [example project](./example_project) about how to add your algorithms easily.
+
+We also provide some documentation listed below:
+
+- [New Model Guide](https://mmpose.readthedocs.io/en/latest/tutorials/4_new_modules.html)
```

Comment:
Should be 1.x documentation

Reply:
This part of doc has not been added in 1.x. The URL for [Model section in Migration](https://mmpose.readthedocs.io/en/1.x/migration.html#step3-model) is used here as an alternative in the latest commit